com.unity.perception/Editor/Randomization/Uss/Styles.uss

```diff
@@ -38,6 +38,11 @@
     background-image: resource("Packages/com.unity.perception/Editor/Icons/ChevronRight.png");
 }
 
+.title-label {

```

Comment:
This class should be prefixed with "scenario__" to follow Unity's USS naming conventions.